.github/workflows/epic-add-to-platform-ux-parent-project.yml

```diff
@@ -0,0 +1,134 @@
+name: When epic issues changed in Platform UX squad porjects, check if epic is part of specified child projects and update on Platform UX parent project
+
+on:
+  issues:
+    types: [opened, edited, reopened, assigned, unassigned, labeled, unlabeled]
```

Comment:
I'm not familiar with GH triggers or the specific cases you'd want to catch - would you care about when issues are closed?

Reply:
That's a good shout. Initially, I was worried about not triggering the workflow too often but since we're filtering for epics only, it should be ok. I will change.